easybuild/easyblocks/generic/configuremake.py

```diff
@@ -83,13 +85,27 @@ def configure_step(self, cmd_prefix=''):
         if prefix_opt is None:
             prefix_opt = '--prefix='
 
-        cmd = "%(preconfigopts)s %(cmd_prefix)s./configure %(prefix_opt)s%(installdir)s %(configopts)s" % {
-            'preconfigopts': self.cfg['preconfigopts'],
-            'cmd_prefix': cmd_prefix,
-            'prefix_opt': prefix_opt,
-            'installdir': self.installdir,
-            'configopts': self.cfg['configopts'],
-        }
+        configure_command = cmd_prefix + './configure'
+        # Avoid using config.guess from an Autoconf generated package as it is frequently out of date, use the version
+        # shipped with EB instead and provide the result to the configure command
+        build_type_option = ''
+        if 'Generated by GNU Autoconf' in open(configure_command).read():
```

Comment:
Hmm, in that case is it safe to assume that with `autotools` will insert a recent config.guess? Otherwise this looks like another reason to take the alternative route.

Reply:
It will use the one provided with the Autotools package (I believe it is the one shipped with Automake).

Reply:
Hmm, seems ok then. In the worst case, we need to patch `Automake` with the latest `config.guess` when adding it to a new toolchain (if they are out of sync).

Reply:
Yes, @geimer is correct, it will use the one provided with the autotools. I'm quite certain it is not the one from Autoconf, so it must be the one from Automake.